cloudlift/deployment/service_template_generator.py

```diff
@@ -220,7 +241,86 @@ def _add_service(self, service_name, config):
             MinimumHealthyPercent=100,
             MaximumPercent=200
         )
-        if 'http_interface' in config:
+        if 'udp_interface' in config:
```

Comment:
i think there is also lot of duplication here, can we extract out common functionality out of the if/else block.